src/components/FormHelpMessage.tsx

```diff
@@ -23,11 +25,29 @@ type FormHelpMessageProps = {
 
     /** Whether to show dot indicator */
     shouldShowRedDotIndicator?: boolean;
+
+    /** Whether should render error text as HTML or as Text */
+    shouldParseText?: boolean;
```

Comment:
How about `shouldParseMessage`? It'd be clear to which prop it's related

Reply:
Not sure but I'm with @war-in on this one, I understand choice of `text` because whole Props are for 'HelpMessage', but we already have `message` attribute so it would match to have `shouldParseMessage`

Reply:
I agree 🤟